content/docs/pipelines/sdk/dsl-overview.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "DSL Overview"
+description = "Introduction to the Kubeflow Pipelines domain-specific language (DSL)"
+weight = 20
++++
+
+The
+[Kubeflow Pipelines DSL](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/dsl)
+is a set of Python libraries that you can use to specify machine learning (ML)
+workflows, including pipelines and their components. (If you're new to
+pipelines, see the conceptual guides to [pipelines](/docs/pipelines/concepts/pipeline/)
+and [components](/docs/pipelines/concepts/component/).)
+
+The
+[DSL compiler](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/compiler)
+compiles your Python DSL code into a single static configuration (YAML)
```

Comment:
nits: code -> codes

Reply:
Thanks! Actually, the singular "code" covers both singular and plural.